docs/HOL_Deploying-Using-Puppet/README.md

```diff
@@ -76,101 +75,28 @@ though _puppet programs_ on the Puppet Master.
     ports and the Parts Unlimited MRP app port on the partsmrp machine. The ARM template opens these ports on the
```

Comment:
image 4.jpg is out of date, the latest version of puppet has a different layout.